sdk/src/Core/Amazon.Runtime/Internal/Util/_netstandard/EncodingExtensions.cs

```diff
@@ -0,0 +1,81 @@
+#if NETSTANDARD
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+using System;
+using System.Text;
+
+namespace Amazon.Runtime.Internal.Util
+{
+    /// <summary>
+    ///     Encoding extensions used to provide a compatibility shim for encoding methods that are missing in .NET Standard
+    ///     2.0.
+    /// </summary>
+    internal static class EncodingExtensions
```

Comment:
Let me know if you prefer another place to put it

Reply:
You can add these extensions method to this .NET Standard extensions file. https://github.com/aws/aws-sdk-net/blob/main/sdk/src/Core/Amazon.Util/_netstandard/Extensions.cs

Reply:
Done